Dockerfile

```diff
@@ -1,2 +1,2 @@
-ARG IMAGE=sentry:9.1.2
-FROM ${IMAGE}-onbuild
+ARG SENTRY_IMAGE
```

Comment:
No, we don't want the default here as we don't want this file to be used w/o `docker-compose`